ACKategories/Classes/Color+Extra.swift

```diff
@@ -108,21 +108,21 @@ public extension UIColor {
 
     /// Create 1 x 1 px image from color.
     public var image: UIImage {
-        return image(CGSizeMake(1, 1))
+        return image(CGSize(width: 1, height: 1))
     }
 
     /// Create image from color with defined size.
-    public func image(size: CGSize) -> UIImage {
-        let rect = CGRectMake(0, 0, size.width, size.height)
+    public func image(_ size: CGSize) -> UIImage {
```

Comment:
haha maybe I should read more than one line next time :D. Anyway, `UIImage.init(color:)` is probably better than this method on UIColor. If we keep this, it should be `func image(ofSize size: CGSize) -> UIImage` (repeating "size" enhances clarity imho)